apollo-portal/src/main/resources/static/i18n/en.json

```diff
@@ -345,6 +347,8 @@
   "Config.ClusterIsDefaultTipContent": "All instances that do not belong to the '{{name}}' cluster will fetch the default cluster (current page) configuration, and those that belong to the '{{name}}' cluster will use the corresponding cluster configuration!",
   "Config.ClusterIsCustomTipContent": "Instances belonging to the '{{name}}' cluster will only fetch the configuration of the '{{name}}' cluster (the current page), and the default cluster configuration will only be fetched when the corresponding namespace has not been released in the current cluster.",
   "Config.HasNotPublishNamespace": "The following environment/cluster has unreleased configurations, the client will not fetch the unreleased configurations, please release them in time.",
+  "Config.RevokeItem.DialogTitle": "Revoke configuration",
+  "Config.RevokeItem.DialogContent": "Modified but unpublished configurations in the current namespace will be revoked. Are you sure to revoke the configuration?",
```

Comment:
How about `Modified but unpublished configurations in the current namespace will be revoked. Are you sure to revoke the configuration changes?`